bin/consusd

```diff
@@ -2,31 +2,32 @@
 
 'use strict';
 
-// Capture an optional port from the command line args
-const PORT = process.argv.reduce((port, arg) => {
-    return (arg.match(/^--port=(\d+)$/) || [0, port])[1];
-}, 80);
-
 var path = require('path');
 var spawn = require('child_process').spawn;
 var fs = require('fs');
+var config = require('config');
+
+// Capture an optional port from the command line args
+const PORT = process.argv.reduce((port, arg) => {
+    return (arg.match(/^--port=(\d+)$/) || [0, port])[1];
+}, config.get('server.port'));
 
 function start() {
     console.log('Attempting to start the Consus server on port ' + PORT + '...');
     try {
-        fs.writeFileSync(path.join(__dirname, '/../.pid'), '', {
+        fs.writeFileSync(config.get('pid_file'), '', {
```

Comment:
I just know that some people like to put pids in a special directory. For instance, they're usually in `/var/run/program-name/pid` in *nix.